From 44998ac03a7816e0f227bfc005d22280db06f0a5 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Tue, 26 Jan 2016 18:49:31 +0000
Subject: [PATCH 0645/1000] MANTIS 0037350 extension: integration of EZ-Access
 device for KIS kiosk

codes.idl
V1.3.2

added DS_TYPES_KEY, DS_TYPES_KEY_DOWN, DS_TYPES_KEY_UP
the values are temporary, until defined in CUSS; see Mantis 0037186: CUSS-Spec: missing definition of DS_TYPES_KEY


git-svn-id: svn://localhost/SelfServiceCommon/trunk@965 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Cuss/codes.idl | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/SelfServiceCommon/Massai/idl/Cuss/codes.idl b/SelfServiceCommon/Massai/idl/Cuss/codes.idl
index cf19fbe5..06c4df0b 100644
--- a/SelfServiceCommon/Massai/idl/Cuss/codes.idl
+++ b/SelfServiceCommon/Massai/idl/Cuss/codes.idl
@@ -25,6 +25,12 @@
 //  @li V1.3.1
 //  @li Mantis 0029957: ConveyorComp: CUSS1.3 Conveyor (without ConveyorAEA)
 //  @li use full CUSS1.3 IDL + Materna extension statuscodes: BAGGAGE_INTRUSION, BAGGAGE_INTRUSION_FRONT, BAGGAGE_INTRUSION_BACK, 
+//
+//  @date 26.01.2016
+//  @li V1.3.2
+//  @li Mantis 0037350: integration of EZ-Access device for KIS kiosk
+//  @li added DS_TYPES_KEY, DS_TYPES_KEY_DOWN, DS_TYPES_KEY_UP
+//  @li the values are temporary, until defined in CUSS; see Mantis 0037186: CUSS-Spec: missing definition of DS_TYPES_KEY
 //                 
 //----------------------------------------------------------------------------
 
@@ -138,6 +144,11 @@ module datastatus
 
     const long DS_TYPES_EPAYMENT                 = 23000; /**< E-Payment: CuPaySchema.xsd */
     const long DS_TYPES_ILLUMINATION             = 24000; /**< LED CUSS_ILLUMINATION.xsd */
+
+    const long DS_TYPES_KEY                      = 25100; /**< key pressed */
+    const long DS_TYPES_KEY_DOWN                 = 25110; /**< key initially pressed */
+    const long DS_TYPES_KEY_UP                   = 25120; /**< key released */
+
 };
 
 /**
-- 
2.41.0.windows.1

